miso-web/src/main/webapp/WEB-INF/pages/editFreezer.jsp

```diff
@@ -17,6 +17,10 @@
   <a id="addingStorageLink" target="_blank" rel="noopener noreferrer">Adding Storage to a Freezer</a>
 </div>
 
+<c:choose>
+      <c:when test="${freezer.isOutOfService()}"><p class="big-warning">Out of Service</p></c:when>
+    </c:choose>
```

Comment:
`choose` is meant for when there are multiple tests. More conventional to use `if` when there's only one